### PR TITLE
Camera chime cluster

### PR DESCRIPTION
**Pull request by gmarcosb**
Split out from https://github.com/project-chip/connectedhomeip/pull/35773:

Add chime cluster from https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10004

Generated with (using https://github.com/project-chip/alchemy/pull/5):
```
 ./alchemy zap --attribute="in-progress"  --sdkRoot=[] --specRoot=[]  'Chime.adoc' 
 ./scripts/tools/zap_regen_all.py
```

### CODE REVIEW

File: src/app/zap-templates/zcl/data-model/chip/chime-cluster.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/Chime.adoc
+Parameters: in-progress 
+Git: 0.7-fall2024-ncr-1158-g413e195c2
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="General"/>
+  <struct name="ChimeSoundStruct" apiMaturity="provisional">
+    <cluster code="0x0556"/>
+    <item fieldId="0" name="ChimeId" type="int8u"/>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/chime-cluster.xml:27]: Spec issue: We usually use "ID", not "Id" for field names.  Please fix the spec, as a followup.

File: src/app/zap-templates/zcl/data-model/chip/chime-cluster.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <domain name="General"/>
+    <name>Chime</name>
+    <code>0x0556</code>
+    <define>CHIME_CLUSTER</define>
+    <description>This cluster provides facilities to configure and play Chime sounds, such as those used in a doorbell.</description>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <attribute code="0x0000" side="server" define="INSTALLED_CHIME_SOUNDS" type="array" entryType="ChimeSoundStruct">InstalledChimeSounds</attribute>
+    <attribute code="0x0001" side="server" define="ACTIVE_CHIME_SOUND_ID" type="int8u" min="0" max="255" default="0" writable="true">ActiveChimeSoundId</attribute>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/chime-cluster.xml:41]: Again, should be "ID", not "Id", but that's a spec issue.

**Review by bzbarsky-apple — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/chime-cluster.xml (hunk 2 above)
Comment (gmarcosb) [reply src/app/zap-templates/zcl/data-model/chip/chime-cluster.xml:41]: Is it worthwhile making a comment[ in the spec PR ](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10004)to fix all these? [I generated this from that branch]

Comment (gmarcosb) [reply src/app/zap-templates/zcl/data-model/chip/chime-cluster.xml]: @aronrosenberg updated the spec & I re-generated using alchemy 👍

**Review by pidarped — APPROVED** [review]

**Pull request merged**